app/front/scripts/components/data-package-api/index.js

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
@borysyuk a small issue, but please turn this off on your IDE, and remove from the files you commit :).